test/test_translator.py

```diff
@@ -228,31 +308,39 @@ def test_get_event_status():
 
     assert test_event_status == valid_event_status
 
+    
+
 
 def test_wzdx_creator() :
-    wzdx_schema = json.loads(open('translator/sample files/validation_schema/wzdx_v3.0_feed.json').read())
-    icone_data =open('translator/sample files/Icone Data/incidents_extended.xml').read()
 
```

Comment:
This is a good start for this method, but we still have quite a few cases to cover here. For instance:
 - info object not included (verify it's created)
 - datafeed_frequency_update there vs not
 - no incidents found
 - etc.

Reply:
i added few edge cases and merged it to dev.